microsoft/surface/common/repos.nix

```diff
@@ -4,8 +4,8 @@
   linux-surface = fetchFromGitHub {
     owner = "linux-surface";
     repo = "linux-surface";
-    rev = "arch-6.6.1-1";
```

Comment:
Is it not better to follow some tags?

Reply:
Sure, can use arch-6.6.4-1. Need to rebuild since this tag is on an earlier commit.